plugins/openstack/pyparts/network.py

```diff
@@ -69,6 +73,17 @@ def get_config_network_info(self):
             config_info['nova'][key] = "{} ({})".format(port.addresses[0],
                                                         port.name)
 
+        for key, port in self.octavia_bind_interfaces.items():
+            if 'octavia' not in config_info:
+                config_info['octavia'] = {}
+
+            if len(port.addresses) > 0:
+                config_info['octavia'][key] = "{} ({})".format(
+                        port.addresses[0], port.name)
```

Comment:
This assumes that the interface will only ever have one address which afaik is fine.

Reply:
Yes